src/components/NewRuntimeModal.js

```diff
@@ -785,6 +792,8 @@ export const NewRuntimeModal = withModalDrawer({ width: 675 })(class NewRuntimeM
                   isClearable: false,
                   onlyInteger: true,
                   value: numberOfPreemptibleWorkers,
+                  disabled: !this.canUpdateNumberOfWorkers(),
+                  tooltip: !this.canUpdateNumberOfWorkers() ? 'Cloud Compute must be in Running status to change preemptibles' : undefined,
```

Comment:
```suggestion
                  tooltip: !this.canUpdateNumberOfWorkers() ? 'Cloud Compute must be in Running status to change number of preemptibles' : undefined,
```

Just to match the language of the other tool tip